specification/authorization/resource-manager/readme.md

```diff
@@ -90,6 +90,15 @@ directive:
     reason: API documentation is not exposed yet. We're making some small changes before publishing documentation.
 ```
 
+
+### Tag: package-preview-2021-04
```

Comment:
Please add suffix "only" to this section 
also create a new "package-preview-2021-04" section similar to [this](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/authorization/resource-manager/readme.md#tag-package-2020-10-01-preview)  
the versions to be referenced should be the latest available version of said API.  
Only add specs referenced on the preview-2020-10 section as other specs are mantained by other teams

Reply:
Added the 'only' suffix. What is the need of another "package-preview-2021-04" section?